dev-docs/e2e/csi-hostpath/README.md

```diff
@@ -0,0 +1,25 @@
+# CSI resources
+
+This directory contains resources for deploying the [csi-driver-host-path].
+
+[csi-driver-host-path]: https://github.com/kubernetes-csi/csi-driver-host-path
+
+## Usage
+
+The `kustomization.yaml` is ready to be applied to our CI runners.
+It will install the CSI driver into the `csi-system` namespace and set up the necessary RBAC resources.
+
+```bash
+kubectl apply -k .
+```
+
+## Provenance
+
+The official deployment flow is a [shell script] and there are no stable resource definitions.
+Therefore, we observed the rendered resources once, at [this commit], and are tracking them in this directory instead.
+
+If necessary, this can be reproduced by logging the directory content of `TEMP_DIR` whenever `kubectl apply` is executed.
+However, we don't expect too many changes here, since the driver isn't released in lockstep with Kubernetes and releases are infrequent.
```

Comment:
The upgrade path still isn't very clear to me. Is this a "never gonna happen anyway" type of upgrade procedure?